### PR TITLE
Draft evaluation for QUIC.

### PR DESCRIPTION
**Pull request by mfoltzgoogle**
Addresses Issue #9: Evaluate QUIC as a protocol transport

Based on conversation with a QUIC developer in Chromium and evaluation of the spec, this is a draft evaluation of QUIC using the template in PR #16.

Many details need to be determined and also reconciled with the current draft specification; my initial review was based on the public documents on chromium.org, and not the current IETF draft.

So consider this a first draft :)

### CODE REVIEW

File: quic.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# QUIC
+
+[QUIC](https://www.chromium.org/quic) is a multiplexed stream transport over
+UDP.  It advertises the following benefits:
```
Comment (chrisn) [inline quic.md]: I suggest changing "benefits" to "benefits over TCP"

File: quic.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+A QUIC connection between client and server transmits data in
+[QUIC Streams](https://tools.ietf.org/html/draft-ietf-quic-transport-00#section-6.1).
+Each Stream represents a bidrectional reliable sequence of data packets, can
+originate from the server or client, and has a unique integer ID.
+
+Assume that a single QUIC connection is used to transmit data between the
+controller and display.  The following mapping could be used to associate
+streams with Presentation API concepts:
+
+- One reserved stream 5 for control messages between the controller and display.
```
Comment (chrisn) [inline quic.md]: "with ID 5" ?

File: quic.md (hunk 1 above)
Comment (mfoltzgoogle) [inline quic.md]: Done.

File: quic.md (hunk 2 above)
Comment (mfoltzgoogle) [inline quic.md]: Done.

**Comment by mfoltzgoogle** [comment]
Thanks for looking it over @chrisn.  Merged

**Pull request merged**